cmd/heznode/README.md

```diff
@@ -13,16 +13,16 @@ NAME:
    hermez-node - A new cli application
 
 USAGE:
-   node [global options] command [command options] [arguments...]
+   heznode [global options] command [command options] [arguments...]
 
 VERSION:
-   v0.1.0-6-gd8a50c5
+   v1.2.0-rc2-5-g94a7043
 
 COMMANDS:
    version    Show the application version
    importkey  Import ethereum private key
-   genbjj     Generate a new BabyJubJub key
-   wipesql    Wipe the SQL DB (HistoryDB and L2DB) and the StateDBs, leaving the DB in a clean state
+   genbjj     Generate a new random BabyJubJub key
+   wipedbs    Wipe the SQL DB (HistoryDB and L2DB) and the StateDBs, leaving the DB in a clean state
```

Comment:
I think we can remove this usage part and keep only the `sync` and `coord` descriptions. It's hard to maintain for each version

Reply:
maybe.

@jeffprestes your opinion?